studio/components/interfaces/Settings/Database/DiskSizeConfiguration.tsx

```diff
@@ -62,7 +60,7 @@ const DiskSizeConfiguration = ({ disabled = false }: DiskSizeConfigurationProps)
     updateProjectUsage({ projectRef, volumeSize })
   }
 
-  const currentDiskSize = projectUsage?.disk_volume_size_gb ?? 0
+  const currentDiskSize = project?.volumeSizeGb ?? 0
```

Comment:
is the volumeSizeGb change already updated on prod?

Reply:
This was already present before actually